packages/classes/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Improved package description.
+- Added time attributes to `Payload`.
+- Added documentation for `Payload`.
```

Comment:
Okay so if we're following Keep A Changelog, these should be nested under a separate `### Added` header, not `### Changed`.

Reply:
Oooooo I should read that!

Fixing.